src/styles/treemap.scss

```diff
@@ -17,6 +17,6 @@
 
 .rv-treemap__leaf__content {
   overflow: hidden;
-  padding: 10px;
+  padding: 4px;
```

Comment:
i think this would be a breaking change, can you over-ride it with local styles?

Reply:
Thank you for warning, I moved this to the local style in d87996e85b43f44596d44a95d1a9dcea2e8372a5